minard/templates/noise.html

```diff
@@ -50,7 +50,7 @@
       <tr>
       {% endif %}
 	<td><a href="{{ url_for('noise_run_detail',run_number = run['run_number']) }}">{{ run["run_number"] }}</a></td>
-	<td>{{ run["run_time"]|timefmt }}</td>
+	<td id="runtime_{{run["run_number"]}}"></td>
```

Comment:
I agree (didn't realize I could embed a call like that). The advantage of the messy way is that the time would render as Sudbury time regardless of where the server is, since as you mentioned below, getting the time right in js is painful. But the page will be served from Sudbury, and that's no worse an assumption than that that nearline process ran in Sudbury, so.